### PR TITLE
BD-1037:Compliance Issues in ThunderClientLibraries

### PR DESCRIPTION
**Pull request by BhargaviPalineni79**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -187,7 +187,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2020 RDK Management
+   Copyright 2020 Metrological
```
Comment (mhughesacn) [inline LICENSE:190]: Please use Copyright [yyyy] [name of copyright owner] here as there is more than one party with an Apache license.

File: NOTICE [hunk 2]
```diff
@@ -7,82 +7,29 @@ listed below. Your use of this material within the component is also subject to
...
 Copyright © 2015-2017 Red Hat Inc.
-Licensed under the MIT license
-
-Copyright (C) 2006-2015, ARM Limited
-Licensed under the Apache License, Version 2.0
+Licensed under the MIT License
 
-Copyright 2014 Fraunhofer FOKUS
+Copyright (C) 2007 The Android Open Source Project
 Licensed under the Apache License, Version 2.0
```
Comment (mhughesacn) [inline NOTICE:33]: My mistake: for consistency, please add:

Copyright (C) 2015,2019 Metrological
Licensed under the BSD-2 License

**Review by mhughesacn — CHANGES_REQUESTED** [review]
Thanks Bhargavi,

Please change the BSD-3 license in LICENSE to BSD-2.

Source/deviceinfo/device_info/CMakeLists.txt
still has an RDK header - please change to Metrological

Plus 2 other comments on individual files.

File: LICENSE [hunk 3]
```diff
@@ -223,7 +224,7 @@
    OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
    SOFTWARE.
 
-   BSD-3 LICENSE
+   BSD-2 LICENSE
```
Comment (mhughesacn) [inline LICENSE:227]: Thanks Bhargavi: you need to delete the third point as well. ("3. Neither the name..." to change the license from BSD-3 to BSD-2.

Comment (BhargaviPalineni79) [reply LICENSE:227]: ok sure

**Comment by mhughesacn** [comment]
Sorry Bhargavi, not sure why I didn't see this earlier:
- Change the RDK copyright in top level CMakelists.txt.  (I think this is the last RDK one.)
- Please do a grep for "this component's license file" and change to "this component's LICENSE file" (just a change of case).  This is in about 20 files.
Thanks,
Martin

**Comment by mhughesacn** [comment]
Thanks Bhargavi.  I updated the BLDKCMF ticket to put Metrological questions in bold.

**Pull request merged**